portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -2029,6 +2029,12 @@ definition {
 
 		Form.editDescription();
 
+		var browserType = PropsUtil.get("browser.type");
+
+		if (("${browserType}" == "internetexplorer") && (IsElementPresent(locator1 = "Form#SIDEBAR_CLOSE"))) {
+			Click(locator1 = "Form#SIDEBAR_CLOSE");
+		}
+
```

Comment:
Is this a consistent issue on CI? Are you able to reproduce locally?

Reply:
I am not able to reproduce on local. On CI, it was consistent before the draganddrop issue happened, but not the draganddrop issue disappeared somehow, and the error changed back to this one.